src/traces/mesh3d/attributes.js

```diff
@@ -57,11 +87,28 @@ module.exports = {
         role: 'style',
         dflt: -1,
         description: [
-            'Sets the shape of the mesh',
-            'If *-1*, Delaunay triangulation is used',
-            'If *>0*, the alpha-shape algorithm is used',
-            'If *0*,  the convex-hull algorithm is used',
-            'An alternative to the `i`, `j`, `k` indices triplets.'
+            'Determines how the mesh surface triangles are derived from the set of',
+            'vertices (points) represented by the `x`, `y` and `z` arrays, if',
+            'the `i`, `j`, `k` arrays are not supplied.',
+            'For general use of `mesh3d` it is preferred that `i`, `j`, `k` are',
+            'supplied, because calculating the surface from points can be very',
+            'time consuming and usually lead to artifacts.',
```

Comment:
I would say shorten this up a bit and rm the explanation "because calculating the surface from points can be very time consuming and usually lead to artifacts."
^^That's true, but we could mention that bit in the blog. Just a suggestion, but I think it might make that description slightly more readable and still informative